go/vt/grpcclient/client.go

```diff
@@ -19,15 +19,24 @@ limitations under the License.
 package grpcclient
 
 import (
+	"flag"
+
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/keepalive"
 
 	"vitess.io/vitess/go/vt/grpccommon"
 	"vitess.io/vitess/go/vt/vttls"
 
 	"vitess.io/vitess/go/vt/log"
 )
 
+var (
+	networkTimeout        = flag.Duration("grpc_network_timeout", 0, "keepalive time and timeout for any network operation in gRPC.")
```

Comment:
For clarity I feel like we should just copy the comments from:
https://github.com/grpc/grpc-go/blob/master/keepalive/keepalive.go#L32-L37

Reply:
I think we should expose separate config flags `keepaliveTime` and `keepaliveTimeout`.
Even if we end up setting them to the same thing, we shouldn't force that policy on everyone.